interfaces/builtin/mount_control.go

```diff
@@ -91,25 +90,74 @@ var allowedMountOptions = []string{
 	"rw",
 }
 
+var allowedUserspaceMountOptions = []string{
+	"auto",
+	"defaults",
+	"noauto",
+	"nofail",
+	"nogroup",
+	"noowner",
+	"nousers",
+}
+
+var allowedFilesystemSpecificMountOptions = map[string][]string{
```

Comment:
I used the `mount(8)` manpage, the manpages for individual filesystems, and source code from `fs/` when neither of the former yielded mount option information. It would be nice if there were an automated way to check for updates to mount options for filesystems.

My thought process was:
1. filesystem options generally won't change much;
2. if they do, they won't be particularly highly-used options;
3. in the off chance a snapd `mount-control` user needs an option which has been newly modified, they should file a bug ask for it so that a security review can happen;
4. this makes the process for changing options similar to that for adding new filesystems altogether (which couldn't really be done automatically), where the new filesystem and options would be added here by hand and reviewed by the security team.

This thought process makes a lot of assumptions, but the effect of forcing manual review is positive while being minimally detrimental to real-world usability, in my opinion.

Reply:
Thanks, that makes perfect sense and I like the comment you added.